maven-uploader-plugin/pom.xml

```diff
@@ -120,6 +120,9 @@
             <plugin>
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-gpg-plugin</artifactId>
+                <configuration>
```

Comment:
I guess you didn't mean to commit skip gpg?

Reply:
right. my mistake.